### PR TITLE
Feat/add form new

### PR DESCRIPTION
**Pull request by borislavkb**
- Add form to Add new item page

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+import { BsPlusSquare } from "react-icons/bs";
 
 export default function Footer() {
   return (
     <footer className="Footer">
       <nav className="Footer__navigation">
         <NavLink to="/">MyCakes</NavLink>
-        <NavLink to="/add">Plus</NavLink>
+        <NavLink to="/add">
+          <BsPlusSquare size="2rem" />
```
Comment (patrikglanemann) [inline src/components/Footer.js:11]: I think inline style is not good practice. You could give it a class or `activeClassName` and style it in your css file.
I found this article on that topic (Method 3):
https://dev.to/ridhikgovind/how-to-style-your-react-router-links-using-styled-components-2350

File: src/pages/AddNewCake.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 1rem;
+  background: var(--color-background);
+}
+
+.btn-submit {
+  background-color: var(--color-text);
+  border: 0px solid white;
```
Comment (patrikglanemann) [inline src/pages/AddNewCake.css:12]: I think what you want to achieve here is that the default border isn't shown.
You could also type this `border: none;`

**Review by patrikglanemann — COMMENTED** [review]
Like to see how it comes slowly together:)
I left you some comments.

File: src/components/Footer.js (hunk 1 above)
Comment (borislavkb) [reply src/components/Footer.js:11]: Thanks! Agree, will check for an alternative approach as I had issues with styling those elements and the icons in particular. :)

**Comment by borislavkb** [comment]
Due to issue with branches the suggested edits will be applied to ```feat/add-icons```branch.

**Pull request closed without merging**